specs/tables.md

```diff
@@ -41,14 +41,14 @@ NOTE:
 Proved by the withdrawal circuit.
 
 This circuit 
-| 0 Withdrawal ID | 1 Validator ID | 2 Address      | 3 Amount      | 4 MPT root     |
-| -----------     | -------------  | -------------- | ------------- | -------------- |
-| $WithdrawalID   | $ValidatorID   | $value{Lo,Hi}  | $value{Lo,Hi} | $value{Lo,Hi}  |
+| 0 Withdrawal ID | 1 Validator ID | 2 Address      | 3 Amount      |
+| -----------     | -------------  | -------------- | ------------- |
+| $WithdrawalID   | $ValidatorID   | $value{Lo,Hi}  | $value{Lo,Hi} |
```

Comment:
Maybe we dont need a full word for the amount if it is u64
```suggestion
| $WithdrawalID   | $ValidatorID   | $value{Lo,Hi}  | $value |
```

Reply:
updated code in [035586d](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/488/commits/035586d031230558551ad74baf198fa9169497e8)